compiler-rt/cmake/builtin-config-ix.cmake

```diff
@@ -162,6 +164,11 @@ if(APPLE)
       list(APPEND DARWIN_tvossim_BUILTIN_ALL_POSSIBLE_ARCHS arm64)
     endif()
   endif()
+  if(COMPILER_RT_ENABLE_XROS)
+    list(APPEND DARWIN_EMBEDDED_PLATFORMS xros)
+    set(DARWIN_xros_BUILTIN_ALL_POSSIBLE_ARCHS arm64)
```

Comment:
Please use `${ARM64} ${ARM32}` similar to what ios does.

Reply:
Done.